server/shared/src/main/scala/org/http4s/server/staticcontent/FileService.scala

```diff
@@ -32,93 +32,158 @@ import org.http4s.server.middleware.TranslateUri
 import org.log4s.getLogger
 import org.typelevel.ci._
 
+import java.io.File
 import scala.util.control.NoStackTrace
 
 object FileService {
   private[this] val logger = getLogger
 
-  type PathCollector[F[_]] = (Path, Config[F], Request[F]) => OptionT[F, Response[F]]
+  @deprecated("use FileService.Fs2PathCollector", "0.23.8")
+  type PathCollector[F[_]] = (File, Config[F], Request[F]) => OptionT[F, Response[F]]
+
+  type Fs2PathCollector[F[_]] = (Path, Config[F], Request[F]) => OptionT[F, Response[F]]
```

Comment:
This was correct before.

Reply:
Thanks! I felt that something was wrong.